.github/ISSUE_TEMPLATE/conduct_violation.yml

```
@@ -0,0 +1,77 @@
+name: Code of Conduct Violation Report
+description: Report a violation of our community Code of Conduct
+title: "Code of Conduct Violation: [Brief Description]"
+labels: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)